app/controllers/internal/AccountTypeController.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.internal
+
+import connector.FrontendAuthConnector
+import controllers.TarRules
+import controllers.internal.AccountType.AccountType
+import engine.RuleEngine
+import model.Locations._
+import model._
+import play.api.Logger
+import play.api.libs.json.{Json, Reads, Writes}
+import play.api.mvc.{Action, AnyContent, Request}
+import services.{ThrottlingService, TwoStepVerification}
+import uk.gov.hmrc.play.frontend.auth.Actions
+import uk.gov.hmrc.play.frontend.controller.FrontendController
+import utils.EnumJson._
+
+object AccountType extends Enumeration {
+  type AccountType = Value
+  val Individual, Organisation = Value
+}
+
+case class AccountTypeResponse(`type`: AccountType)
+
+object AccountTypeResponse {
+  implicit val userTypeReads = enumFormat(AccountType)
+  implicit val writes: Writes[AccountTypeResponse] = Json.writes[AccountTypeResponse]
+  implicit val reads: Reads[AccountTypeResponse] = Json.reads[AccountTypeResponse]
+}
+
+object AccountTypeController extends AccountTypeController {
+  override protected def authConnector = FrontendAuthConnector
+
+  override val defaultLocation = BusinessTaxAccount
+
+  override val ruleEngine = TarRules
+
+  override val throttlingService = ThrottlingService
+
+  override val twoStepVerification = TwoStepVerification
+
+  override def createAuditContext() = AuditContext()
```

Comment:
About omission of parentheses from http://docs.scala-lang.org/style/method-invocation.html

> However, this syntax should only be used when the method in question has no side-effects (purely-functional). In other words, it would be acceptable to omit parentheses when calling queue.size, but not when calling println(). This convention mirrors the method declaration convention given above.

> Religiously observing this convention will dramatically improve code readability and will make it much easier to understand at a glance the most basic operation of any given method. Resist the urge to omit parentheses simply to save two characters!